.code-samples.meilisearch.yaml

```diff
@@ -152,6 +152,17 @@ update_settings_1: |-
   synonyms.insert(String::from("wolverine"), vec!["xmen", "logan"]);
   synonyms.insert(String::from("logan"), vec!["wolverine"]);
 
+  let min_word_size_for_typos = MinWordSizeForTypos {
+    one_typo: Some(4),
+    two_typos; Some(12)
+  }
+  let typo_tolerance = TypoToleranceSettings {
+    enabled: Some(true),
+    disable_on_attributes: Some(vec!["title".to_string()]),
+    disable_on_words: Some(vec![])
```

Comment:
```suggestion
```

The fields that are not required should not be added to the samples. Normally this should not raise an error, right?